INSTALL.md

```diff
@@ -117,18 +117,26 @@ rpm -i oracle-instanclient*
 yum install gcc php-pear php-devel
 ```
 
-Set the pear http proxy if necessary, and the install the oci8 module using pecl:
+Optionally, set the pear http proxy if necessary:
 
 ```bash
 pear config-set http_proxy http://pro.xy:port
-pecl install oci8-2.0.10
 ```
 
+Install the oci8 module using pecl:
+
+```bash
+pecl install oci8-2.0.10
+```
 This will download and compile the module, and place it in your php extension dir.
 
-Add the ```extension=oci8.so``` line to your php.ini. Confirm it is working with ```php -i | grep -i oci8```
+Add the ```extension=oci8.so``` line to your php.ini or create a configuration file: 
```

Comment:
Actually - I would suggest there is no or. Users _should_ be creating a seperate  ini configuration file.

Reply:
Did you not want this changed before merging?